### DISCUSSION TITLE
Security Onion error pvcreate

### DISCUSSION THREAD

**Discussion by vdr0011** [Security-Onion-Solutions/securityonion#10034] [Unsupported Versions]
Has anyone seen this error when trying to install Security Onion VM Workstation 17
![Screenshot 2023-03-28 101921](https://user-images.githubusercontent.com/128842871/228289312-c700f9c0-9560-428a-92bc-7f17b5aa919f.png)

**Comment by cm-ops** [comment]
Duplicate to https://github.com/Security-Onion-Solutions/securityonion/discussions/10035
